### PR TITLE
[BEAM-5124] Documentation update.

### PR DESCRIPTION
**Pull request by VaclavPlajt**
Written on top of your docs update.

Once your pull request has been opened and assigned a number, please edit the
URL below, replacing `PULL_REQUEST_NUMBER` with the number of your pull request.

http://apache-beam-website-pull-requests.storage.googleapis.com/PULL_REQUEST_NUMBER/index.html

Finally, it will help us expedite review of your Pull Request if you tag
someone (e.g. @username) to look at it.

### CODE REVIEW

File: src/documentation/sdks/euphoria.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Dataset<String> output =
         .using(p -> p.getKey() + ": " + p.getValue())
         .output();
 
-// Transform Dataset back to PCollection. It can be done in any step of this flow.
+// Transform Dataset back to PCollection. It can be done any time.
```
Comment (mareksimunek) [inline src/documentation/sdks/euphoria.md]: probably together anytime, its adverb here

File: src/documentation/sdks/euphoria.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ PCollection<T> collection = dataset.getPCollection();
 ```
 
 ### Inputs and Outputs
-Input data can supplied through Beams IO into `PCollection`, the same way as in Beam, and and wrapped by `Dataset.of(PCollection<T> pCollection)` into `Dataset` later on.
+Input data can supplied through Beams IO into `PCollection`, the same way as in Beam, and wrapped by `Dataset.of(PCollection<T> pCollection)` into `Dataset` later on.
```
Comment (mareksimunek) [inline src/documentation/sdks/euphoria.md]: can `be` supplied?

**Review by mareksimunek — CHANGES_REQUESTED** [review]

File: src/documentation/sdks/euphoria.md (hunk 1 above)
Comment (VaclavPlajt) [reply src/documentation/sdks/euphoria.md]: right ;-)

File: src/documentation/sdks/euphoria.md (hunk 2 above)
Comment (VaclavPlajt) [reply src/documentation/sdks/euphoria.md]: thanks

**Pull request merged**